<launch>
  <node name="test_octomap" pkg="octomap_server" type="octomap_server_node" args="$(find vfh_rover)/fr_079.bt">
    <param name="max_depth" value="13" />
  </node>

  <param name="/init_x" value="-5" />
  <param name="/init_y" value="0" />
  <param name="/init_z" value="1" />
  <param name="/height" value="1" />
  <param name="/width" value="1" />
  <param name="/depth" value="1" />
  <param name="/safety_radius" value="0.4" />
  <param name="/maxIncline" value="M_PI/8" />
  <param name="/minIncline" value="-M_PI/8" />
  <param name="/prevSet" value="false" />

  <node name="vfh" pkg="vfh_rover" type="vfh_rover_node"/>
  <node name="goal_pub" pkg="vfh_rover" type="goal_pub.py"/>
  <node name="rover_sim" pkg="vfh_rover" type="simRover.py"/>
</launch>
